numpy/core/src/multiarray/ctors.c

```diff
@@ -1629,16 +1629,14 @@ _array_from_buffer_3118(PyObject *memoryview)
  *              validate and possibly copy arr itself ...
  *      }
  *      ... use arr ...
- * context is passed to PyArray_FromArrayAttr, which ignores it. Since this is
- * a NUMPY_API function, we cannot remove it.
  */
 NPY_NO_EXPORT int
-PyArray_GetArrayParamsFromObject(PyObject *op,
+PyArray_GetArrayParamsFromObject_int(PyObject *op,
                         PyArray_Descr *requested_dtype,
                         npy_bool writeable,
                         PyArray_Descr **out_dtype,
                         int *out_ndim, npy_intp *out_dims,
-                        PyArrayObject **out_arr, PyObject *context)
+                        PyArrayObject **out_arr)
```

Comment:
line 1739: context no longer is defined, the call should replace `context` with `NULL`